npl/src/main/npl/iou/iou.npl

```diff
@@ -13,6 +15,10 @@ notification RepaymentOccurrence(paymentAmount: Number, remainingAmount: Number)
 protocol[issuer, payee] Iou(var description: Text, var forAmount: Number) {
     require(forAmount > 0, "Initial amount must be strictly positive");
 
+    init {
```

Comment:
Could we find something a bit more meaningful here? As this also goes to external users and this seems to be rather ad hoc demonstration of this

Reply:
Reworked as suggested.